.circleci/config.yml

```diff
@@ -23,22 +23,23 @@ aliases:
       name: Calculate cache key
       command: |
         if [ -n "$CIRCLE_PR_NUMBER" ]; then
-          md5sum .circleci/config.yml cabal.project .ci/cabal.project.local */*.cabal > .circleci_cachekey
+          date +%Y-%m > .circleci_cachekey
```

Comment:
I don't think `date` is available on nixos before restoring the cache / running the setup step.

Reply:
Good point
But it's there:
```
$ docker run -it nixos/nix
Unable to find image 'nixos/nix:latest' locally
latest: Pulling from nixos/nix
9d48c3bd43c5: Pull complete 
480223332b13: Pull complete 
74bc3bbbef74: Pull complete 
Digest: sha256:5d0a26066dc8e9e1295676203cbc15d0e124f445007f9cb888fde25c3d00a1b2
Status: Downloaded newer image for nixos/nix:latest
362e45a9967c:/# date
Thu Feb 27 14:34:11 UTC 2020
362e45a9967c:/# date +%Y-%m
2020-02
```